frontend/src/pages/learn/when-to-take-your-pensions.tsx

```diff
@@ -0,0 +1,297 @@
+      </h2>
+      <p>{t('old-age-security.content-one')}</p>
+
+      <div className="flex justify-center">
+        <p>
+          {t('old-age-security.table-one.title')}
+        </p>
+      </div>
+
+      <div className="overflow-x-auto">
+        <table className="min-w-full divide-y divide-[#B9B9B9] border border-[#B9B9B9] border-collapse">
+          <thead className="bg-gray-50">
+            <tr>
+              <th className="w-1/3 px-6 py-3 text-left text-xs font-strong text-black tracking-wider border-r border-[#B9B9B9]">
+                {t('old-age-security.table-one.header-one')}
+              </th>
+              <th className="w-1/3 px-6 py-3 text-left text-xs font-strong text-black tracking-wider border-r border-[#B9B9B9]">
+                {t('old-age-security.table-one.header-two')}
+              </th>
+              <th className="w-1/3 px-6 py-3 text-left text-xs font-strong text-black tracking-wider">
+                {t('old-age-security.table-one.header-two')}
+              </th>
+            </tr>
+          </thead>
+          <tbody className="bg-white divide-y divide-[#B9B9B9]">
+            {[{
+              col1: t('old-age-security.table-one.row-one.column-one-value'),
+              col2: t('old-age-security.table-one.row-one.column-two-value'),
+              col3: t('old-age-security.table-one.row-one.column-three-value')
+            },
+            {
+              col1: t('old-age-security.table-one.row-two.column-one-value'),
+              col2: t('old-age-security.table-one.row-two.column-two-value'),
+              col3: t('old-age-security.table-one.row-two.column-three-value')
+            }].map((row, index) => (
+              <tr key={index} className={`${index === 0 ? 'bg-[#B9B9B9]' : ''}`}>
+                <td className="px-6 py-4 text-xs font-medium text-black whitespace-nowrap border-r border-[#B9B9B9] bg-white">{row.col1}</td>
+                <td className="px-6 py-4 text-xs font-medium text-black whitespace-nowrap border-r border-[#B9B9B9] bg-white">{row.col2}</td>
+                <td className="px-6 py-4 text-xs font-medium text-black whitespace-nowrap bg-white">{row.col3}</td>
+              </tr>
+            ))}
+          </tbody>
+        </table>
+      </div>
+      <ImportantCard>
+        <Trans
+          ns="learn/when-to-take-your-pensions"
+          i18nKey="old-age-security.smart-tip.content"
+          components={{
+            a1: (
+              <a className="underline"
+                href={t(
+                  'old-age-security.smart-tip.link-one'
+                )}
+              />
+            ),
+            a2: (
+              <a className="underline"
+                href={t(
+                  'old-age-security.smart-tip.link-two'
+                )}
+              />
+            )
+          }}
+        />
+      </ImportantCard>
+      <p>{t('old-age-security.content-two')}</p>
+      <div className="w-full lg:flex-1">
```

Comment:
A block-level element is an HTML element that takes up horizontal space by default. It stretches 100% wide and takes on the height of its inner content. Do the same for other divs with same classes.
```suggestion
      <div>
```